.github/workflows/build.yaml

```diff
@@ -0,0 +1,54 @@
+name: Build
```

Comment:
We are currently maintaining a set of reusable workflows for our charms. I think you might be interested in taking a look at them: https://github.com/canonical/operator-workflows :)

Reply:
I created a ticket in out backlog to discover this repo